docs/guide/targetgroupbinding/targetgroupbinding.md

```diff
@@ -17,6 +17,13 @@ TargetGroupBinding CR supports TargetGroups of either `instance` or `ip` TargetT
     If TargetType is not explicitly specified, a mutating webhook will automatically call AWS API to find the TargetType for your TargetGroup and set it to correct value.
 
 
+## VpcId
+TargetGroupBinding CR supports the explicit definition of the Virtual Private Cloud (VPC) of your TargetGroup.
+
+!!!tip ""
+    If the VpcId is not explicitly specified, a mutating webhook will automatically call AWS API to find the VpcId for your TargetGroup and set it to correct value.
+
```

Comment:
Can you please also add a sample yaml file here?

Reply:
comment addressed